wsidicom/instance.py

```diff
@@ -57,20 +58,39 @@ class Requirement(IntEnum):
 @dataclass
 class WsiAttribute:
```

Comment:
Does this class really represent a WsiAttribute? It looks like it has to do with if a attribute is required or not. Rename to `WsiAttributeRequirement` or something like that and/or add docstrings explain what it's purpose is.

Reply:
Renamed and added docstring.